doc/manual/manual.tex

```diff
@@ -6730,7 +6731,72 @@ \subsubsection{Crustal deformation}
 \label{fig:ext3D}
 \end{figure}
 
+\subsubsection{Continental extension}
+\label{sec:cookbooks-continental-extension}
+\textit{This section was contributed by John Naliboff}
 
+In the crustal deformation examples above, the viscosity depends solely on the drucker prager yield criterion defined by the cohesion and internal friction angle.  While this approximation works reasonably well for the uppermost crust, deeper portions of the lithosphere may undergo either brittle or viscous deformation, with the latter depending on a combination of composition, temperature, pressure and strain-rate.  In effect, a combination of the Drucker-Prager and Diffusion dislocation material models is required.  The visco-plastic material model is designed to take into account both brittle (plastic) and non-linear viscous deformation, thus providing a template for modeling complex lithospheric processes. Such a material model can be used in \aspect{} using the following set of input parameters: 
```

Comment:
Ah, just saw that this is your way of separating sentences. If so, and this does not show up in the document that is totally fine.